src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
I am sorry, I said it wrong... it replaces it by app.url in your case because the app.url value in web.xml and test.url in test.properties are the same. In general they dont have to be the same.